src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -201,8 +196,9 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
     }
 
     /**
+     * Returns true if there is some open or published email sent for the course.
+     *
```

Comment:
Don't we agree that there should be an empty line between description and at clauses (although not enforced)?